iphone/Classes/TiContactsPerson.m

```diff
@@ -948,28 +948,28 @@ -(void)setValue:(id)value forUndefinedKey:(NSString*)key
 //For iOS9 deleting contact
 -(CNSaveRequest*)getSaveRequestForDeletion
 {
-	CNSaveRequest *saveRequest = [[CNSaveRequest alloc] init];
+	CNSaveRequest *saveRequest = [[[CNSaveRequest alloc] init] autorelease];
```

Comment:
same thing here.
https://github.com/appcelerator/titanium_mobile/pull/7464/files
We cannot do these autorelease.